src/locale/messages.fr.xlf

```diff
@@ -35,65 +35,47 @@
           <context context-type="linenumber">14,15</context>
         </context-group>
       </trans-unit><trans-unit id="5402924068010392760" datatype="html">
-        <source>Join the group "<x id="PH" equiv-text="response.group.name"/>"</source><target state="translated">Rejoindre le groupe "<x id="PH" equiv-text="response.group.name"/>"</target>
+        <source>Join the group "<x id="PH" equiv-text="        header: $lo"/>"</source><target state="translated">Rejoindre le groupe "<x id="PH" equiv-text="response.group.name"/>"</target>
```

Comment:
this change seems strange and unrelated
same for all this file...

Reply:
running `npm run extract-i18n` does that on my computer so I don't really know what to do

Reply:
Without changing anything by hand, just running `npm run extract-i18n`? If you have this `        header: $lo` for id="5402924068010392760" in the `messages.xlf`,  it is strange. Do you have version 11 of `ng` installed (`ng --version`)?

Anyway, you should never push something which is not related in a PR.

Reply:
I've removed this whole file and I will let you do the translation.